target/linux/kirkwood/image/Makefile

```diff
@@ -57,6 +57,16 @@ define Device/cloudengines_pogoplugv4
 endef
 TARGET_DEVICES += cloudengines_pogoplugv4
 
+define Device/checkpoint_l50
+  DEVICE_VENDOR := CheckPoint
+  DEVICE_MODEL := L-50
```

Comment:
Strictly, when model name is "L-50", the device name and DTS name should be "checkpoint_l-50".
However, I do not know whether anybody cares about that on kirkwood.

Reply:
I can't see any names convention in kirkwood.
Are @p-wassi still kirkwood maintainer?

Reply:
I do not think there is one. ;-)
Since I'm currently trying to harmonize names OpenWrt-wide, I'd prefer the version with the hyphen, as this would match what I require for targets where I'm more involved, like ath79 and ramips. But I won't force you. Note that in this case you would have to adjust DTS name and compatible as well.

Reply:
the maintainer mentioned in the file ( @lperkov ) has been inactive for years

I (as one of the contributors that added kirkwood devices and did some maintenance on the target) would personally prefer "checkpoint_l-50" too

Reply:
@adrianschmutzler @bobafetthotmail 
I will add hypen.

OT:
"MAINTAINER" field isn't actual in OpwnWRT. At this moment 14 targets have at least  unactive maintainers. 


````./target/linux/mpc85xx/Makefile:14:MAINTAINER:=Imre Kaloz <kaloz@openwrt.org> # last commit 2018
./target/linux/mvebu/Makefile:13:MAINTAINER:=Imre Kaloz <kaloz@openwrt.org> # last commit 2018
./target/linux/imx6/Makefile:15:MAINTAINER:=Luka Perkov <luka@openwrt.org> # one commit in 2019, 0 in 2018
./target/linux/kirkwood/Makefile:14:MAINTAINER:=Luka Perkov <luka@openwrt.org> # one commit in 2019, 0 in 2018
./target/linux/gemini/Makefile:14:MAINTAINER:=Roman Yeryomin <roman@advem.lv> # four commits in 2019
./target/linux/zynq/Makefile:15:MAINTAINER:=Jason Wu <jason.wu.misc@gmail.com> # last commit 2016
./target/linux/samsung/Makefile:14:MAINTAINER:=Jianhui Zhao <jianhuizhao329@gmail.com> # last commit 2018 
./target/linux/ath25/Makefile:13:MAINTAINER:=Sergey Ryazanov <ryazanov.s.a@gmail.com> # last commit 2018
./target/linux/octeontx/Makefile:14:MAINTAINER:=Tim Harvey <tharvey@gateworks.com> #active?
./target/linux/at91/Makefile:11:MAINTAINER:=Claudio Mignanti <c.mignanti@gmail.com> \ #last commit 2012
./target/linux/rb532/Makefile:13:MAINTAINER:=Roman Yeryomin <roman@advem.lv> # four commits in 2019
./target/linux/arc770/Makefile:12:MAINTAINER:=Alexey Brodkin <abrodkin@synopsys.com> # last commit 2017
./target/linux/apm821xx/Makefile:12:MAINTAINER:=Chris Blake <chrisrblake93@gmail.com>, \ # three commits in 2019
./target/linux/archs38/Makefile:13:MAINTAINER:=Alexey Brodkin <abrodkin@synopsys.com> # last commit 2017
````

Reply:
Well, if you look at the targets, most of them aren't very "active" themselves. So, it might be hard to find a replacement.

BTW: "actual" means something like "in fact", while you most probably meant "up-to-date" instead ... Looks like Polish language is similarly unhelpful here as German with "aktuell".

Reply:
Hypen added.
My plan for this target after this PR is switch to 4.19 (it works well) and switch to generic diag.sh.


> Well, if you look at the targets, most of them aren't very "active" themselves. So, it might be hard to find a replacement.

Maybe better approach is blank field instead ghost maintainer?

>BTW: "actual" means something like "in fact", while you most probably meant "up-to-date" instead ... Looks like Polish language is similarly unhelpful here as German with "aktuell".

Exactly. My bad.